crates/driver/src/tests/setup/mod.rs

```diff
@@ -380,6 +380,22 @@ impl Solution {
         }
     }
 
+    pub fn increase_gas(self, units: usize) -> Self {
+        // non-zero bytes costs 16 gas
+        let additional_bytes = units / 16;
```

Comment:
This division truncates the fraction so it might be that you are not increasing the gas amount at all.
The options I see to handle this:
1. panic if you pass a value that is not a multiple of 16
2. unless `units == 0` add 1 extra byte to compensate for the truncation and mention that it increases the gas by at least `units`
3. add a doc comment explaining the behavior (should probable be done regardless)